CHANGELOG.md

```diff
@@ -26,7 +26,9 @@ CHANGELOG
 ------
 **ENHANCEMENTS**
 
-- Enable support for ARM instances in China and GovCloud regions when using Ubuntu 18.04 or Amazon Linux 2. 
+- Enable support for ARM instances in China and GovCloud regions when using Ubuntu 18.04 or Amazon Linux 2.
+- Add validation for `cluster_type` configuration parameter in `cluster` section
```

Comment:
We might want to backport these changelog changes to 2.10 branch

Reply:
Looks like they're already there: https://github.com/aws/aws-parallelcluster/blob/release-2.10/CHANGELOG.md.
We probably forgot to update the develop version.